staging/src/k8s.io/client-go/rest/request.go

```diff
@@ -125,6 +126,10 @@ type Request struct {
 	bodyBytes []byte
 
 	retryFn requestRetryFunc
+
+	// gctx defined the global context, it's set by method chaining Context().
+	// We can get it using the r.ctx() member function, which ensures that returns context is always not nil
+	gctx context.Context
```

Comment:
The context isn't global? It's specific for the request. I would simply call the field `ctx`.

Reply:
Done, thanks.